README.md

```diff
@@ -43,6 +43,8 @@ received on the response of the top-level navigation, over a secure connection.
 They can similarly opt-in by using the headers' HTML equivalents, the `<meta>`
 HTML tag and its `http-equiv` attribute.
 
+[TODO: opt-in is double keyed]
```

Comment:
Since the opt-in is only allowed on the top-level navigation response, I think there's no real need to add double-keying protections: https://whatpr.org/html/3774/e32a6f8...335f566/browsing-the-web.html#initialise-the-document-object